_src/static/sass/main-rtl.scss

```diff
@@ -13,14 +13,19 @@
 @import '../../vendor/bi-app/bi-app-rtl';   // RTL support
```

Comment:
This compile structure and some of the file/folder organization does not match our planned for compile order in https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#main-style-compile. Please sync up with it and with main-ltr.scss.